src/ceph-daemon/ceph-daemon

```diff
@@ -246,7 +246,7 @@ def infer_fsid(func):
 
         if not fsid_list:
             # TODO: raise?
```

Comment:
minor, but maybe it's cleaner to raise here ..
```
raise RunetimeError('must pass --fsid to specify cluster')
```

instead of needing to check for `args.fsid` in each instance where `@infer_fsid` is used?